### ISSUE TITLE
Adapt app-frontend-react to accept reload commands from Studio

### ISSUE THREAD

**Issue by standeren** [Altinn/app-frontend-react#1088]
### Description

Studio is implementing a preview functionality where app-frontend is rendered inside an iframe and to enable hot reload it is desirable that app-frontend adapts to resend som calls to app-backend-api which is mocked in Studio backend to get updated versions of config files.

Studio will ask app-frontend to trigger new calls to the following endpoints:
- `.../{org}/{app}/api/resource/{formLayoutName}`
- `.../{org}/{app}/api/v1/texts/{languageCode}`
- `.../{org}/{app}/api/layoutsettings`

These messages will be sent to app-frontend with `contentWindow.postMessage("action: reload-[config]", "[baseurl-of-env-studio-runs]")`

### Additional Information

To separate the calls to the mocked api from the actual app-backend-api it is also desirable that the api-route used for the mocked api ends with the suffix `preview`, resulting in; `{org}/{app}/preview/...`

**Comment by standeren** [comment]
Preferably all 😄 And I regret calling it a prefix, I meant suffix; `{org}/{app}/preview`

**Comment by standeren** [comment]
Is it also possible to adapt this special api (and frontend?) so that it is not possible to press the user/organization logo and get the option to _log out_? We can of course mock the behaviour of this action so nothing happens, but it would be more user friendly if you didn't get the option at all 🤷‍♀️

**Comment by olemartinorg** [comment]
@standeren Yup, we could probably arrange that. In the hackathon, I made sure to set a property in the `contentWindow` object before the app loads (if I recall correctly) so that the app knows that it is rendered in preview mode when first loading. We could hide the logout button if that's the case.
